sklearn/ensemble/_iforest.py

```diff
@@ -509,6 +522,26 @@ def _more_tags(self):
         }
 
 
+def _calculate_depths(tree):
+    """
+    Calculates the depths of each node in a tree.
```

Comment:
Nitpick on docstring formatting.

```suggestion
    """Calculate the depths of each node in a tree.

```

also: "calculate" sounds weird to me and I would have expected "compute" but I am not a native English speaker so maybe it's completely fine.

Reply:
I agree, renamed to `_compute_depths`.